Mage.Sets/src/mage/sets/Battlebond.java

```diff
@@ -9,23 +9,27 @@
  * @author TheElk801
  */
 public final class Battlebond extends ExpansionSet {
-
+    
     private static final Battlebond instance = new Battlebond();
-
+    public int i;
+    
     public static Battlebond getInstance() {
         return instance;
     }
-
-    private Battlebond() {
```

Comment:
This constructor should remain private as it is intended to be a singleton class.